lib/membrane/rtp/serializer.ex

```diff
@@ -84,7 +84,7 @@ defmodule Membrane.RTP.Serializer do
       timestamp: rtp_timestamp,
       sequence_number: state.sequence_number,
       csrcs: Map.get(rtp_metadata, :csrcs, []),
-      extension: Map.get(rtp_metadata, :extension)
+      extensions: [Map.get(rtp_metadata, :extension)]
```

Comment:
Will this need further refactor in other payloaders to return the list or are we fine with returing just one extension from them?

Reply:
This was a bug, It will look like this now 
```elixir
extensions: Map.get(rtp_metadata, :extensions)
```